src/web/app/components/question-types/question-edit-answer-form/constsum-options-question-edit-answer-form.component.html

```diff
@@ -5,7 +5,7 @@
   <div class="col-12 col-sm-5">
     <input type="number" class="form-control"
            [ngModel]="i < responseDetails.answers.length ? responseDetails.answers[i] : ''"
-           (ngModelChange)="triggerResponse(i, $event)"
+           (ngModelChange)="triggerResponse(i, Math.ceil($event))"
```

Comment:
Shouldn't this operation be performed by the receiving component instead of the event emitter?

Reply:
Depends on the responsibility. I would say it is the form's responsibility to ceil the number.

Reply:
Ok I put the `Math.ceil()` within `triggerResponse` since it is already processing the data, and it doesn't make sense to process data in 2 different places.